services/QuillCMS/Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'dotenv-rails', '~> 2.7'
 gem 'activerecord-import'
 gem 'bulk_insert'
 gem 'pg', '~> 1.2'
+gem 'scenic'
```

Comment:
Can we pin this to `~> 1.5.4`? This will make re-bundling easier in the future.

Reply:
Another comment that was not added to my review: there is one GradedResponse spec that is failing